ui/desktop/app/styles/app.scss

```diff
@@ -512,12 +512,16 @@ $chevron-down: url("data:image/svg+xml;utf-8,<svg viewBox='0 0 24 24' fill='%231
   }
 }
 
-.details-sidebar-target-details {
+.details-sidebar {
   background-color: var(--token-color-palette-neutral-50);
   border-radius: 0.313rem;
   padding: 1.5rem 2.5rem;
   margin-top: 3.5rem;
 
+  .detail-group:first-child {
```

Comment:
Doesn't this add margin if there's only one child, e.g. in the case where we just show a target with no host details?

Reply:
Yeah. Let me update this.

Reply:
Fixed.